src/bundle/Resources/public/css/modules/modal.css

```diff
@@ -0,0 +1,31 @@
+.ez-modal {
+    position: fixed;
+    padding: 0;
+    border: 0;
+    border-radius: 8px;
+    background: var(--ez-color-ground-modal);
+
+    box-shadow: 0 3px 20px var(--ez-color-shadow-modal);
+}
+
+.ez-modal header, .ez-modal section, .ez-modal footer {
+    padding: 1em;
+}
+
+.ez-modal header {
```

Comment:
@SylvainGuittard @yannickroger it's not a left over, it's made on purpose because `.ez-modal` class will be used for others modals than the confirm dialog in the app and [the UI Guidelines specifies that a modal can have a `<header>`](https://ezsystems.github.io/ui-guidelines/#modal3)